packages/appstore/src/components/banners/book-banner/book-banner.tsx

```diff
@@ -0,0 +1,58 @@
+import React from 'react';
+import { SessionStore } from '@deriv/shared';
+import { getAllowedLanguages, getLanguage } from '@deriv/translations';
+import { Analytics } from '@deriv-com/analytics';
+import BookBannerTemplate from 'Components/banners/book-banner/banner-template';
+
+type TEbooks = 'forex-ebook' | 'stock-ebook' | 'cryptocurrencies-ebook' | 'synthetic-indices-ebook' | 'chart-patterns';
+export type TEbooksUrl = { [B in TEbooks]: Record<keyof ReturnType<typeof getAllowedLanguages>, string> };
+
+const e_books_url: TEbooksUrl = {
+    'forex-ebook': {
+        EN: 'https://deriv.link/ebook-forex-en-lq',
+        FR: 'https://deriv.link/ebook-forex-fr-lq',
+        PT: 'https://deriv.link/ebook-forex-pt-lq',
+        ES: 'https://deriv.link/ebook-forex-sp-lq',
+        VI: 'https://deriv.link/ebook-forex-vn-lq',
+    },
+    'stock-ebook': {
+        EN: 'https://deriv.link/ebook-stocks-en-lq',
+        FR: 'https://deriv.link/ebook-stocks-fr-lq',
+        PT: 'https://deriv.link/ebook-stocks-pt-lq',
+        ES: 'https://deriv.link/ebook-stocks-sp-lq',
+    },
+    'cryptocurrencies-ebook': {
+        EN: 'https://deriv.link/ebook-crypto-en-lq',
+        FR: 'https://deriv.link/ebook-crypto-fr-lq',
+        PT: 'https://deriv.link/ebook-crypto-pt-lq',
+        ES: 'https://deriv.link/ebook-crypto-fr-lq',
+    },
+    'synthetic-indices-ebook': {
+        EN: 'https://deriv.link/ebook-synthetics-en-lq',
+        FR: 'https://deriv.link/ebook-synthetics-fr-lq',
+        PT: 'https://deriv.link/ebook-synthetics-pt-lq',
+        ES: 'https://deriv.link/ebook-synthetics-sp-lq',
+        VI: 'https://deriv.link/ebook-synthetics-vn-lq',
+    },
+    'chart-patterns': {
+        EN: 'https://deriv.link/ebook-10chart-en-lq',
+        FR: 'https://deriv.link/ebook-10charts-fr-lq',
+        PT: 'https://deriv.link/ebook-10charts-pt-lq',
+        ES: 'https://deriv.link/ebook-10charts-sp-lq',
+    },
+};
+const e_book_show_way: unknown = Analytics.getFeatureValue('e_book', 'inactive');
+
+const BookBanner = () => {
+    const e_book_from_landing: TEbooks = SessionStore.get('show_book');
+    const lang = getLanguage();
+
+    if (e_book_from_landing && e_book_show_way === 'banner') {
```

Comment:
better check for negation of this condition and return null first
```suggestion
    if (!e_book_from_landing || e_book_show_way !== 'banner') {
    return null }
    ....
```

Reply:
controversial decision in terms of cognitive load, but corrected

Reply:
I changed my mind. I'll not doing like that because in the future we'll add more types of banner here, let's check commented code